docs/how-to-guides/deploy-feathr-api-as-webapp.md

```diff
@@ -0,0 +1,108 @@
+# Feathr API
+The API currently supports following functionality
+
+1. Get Feature by Qualified Name
+2. Get Feature by GUID
+3. Get List of Features
+4. Get Lineage for a Feature
+
+
+## Build and run locally
+### Install
+
+You can install dependencies through the requirements file
+```bash
+$ pip install -r requirements.txt
+```
+
+### Run
+This command will start the uvicorn server locally and will dynamically load your changes.
+```bash
+uvicorn api:app --port 8080 --reload
```

Comment:
what's the license for uvicorn?

Reply:
https://github.com/encode/uvicorn/blob/master/LICENSE.md

It is a BSD License, and redistribution is permitted as long as we retain their copyright notice.

Reply:
Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:

Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer.

Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimer in the documentation and/or other materials provided with the distribution.

Neither the name of the copyright holder nor the names of its contributors may be used to endorse or promote products derived from this software without specific prior written permission.